opus/application/test_api/responses/api_metadata_vg_iss_2_s_c4360845_default_html.html

```diff
@@ -97,8 +103,8 @@
 <dt>Observed Incidence Angle (Max) (degrees)</dt><dd>94.396</dd>
 <dt>Observed Emission Angle (Min) (degrees)</dt><dd>0.219</dd>
 <dt>Observed Emission Angle (Max) (degrees)</dt><dd>88.886</dd>
-<dt>Observed Solar Hour Angle (Min) (degrees)</dt><dd>0</dd>
-<dt>Observed Solar Hour Angle (Max) (degrees)</dt><dd>360</dd>
+<dt>Observed Local Hour Angle (Min) (degrees)</dt><dd>0</dd>
```

Comment:
Why did this change?

Reply:
Under <target> surface geometry (in obs_suraface_geometry_target.json), the label_results for "solar_hour_angle1/2" fields are "Observed Local Hour Angle (Min/Max)"